src/mmw/js/src/draw/views.js

```diff
@@ -56,21 +56,32 @@ var ToolbarView = Marionette.LayoutView.extend({
     },
 
     onShow: function() {
-        this.selectTypeRegion.show(new SelectAreaView({
-            model: this.model
-        }));
-        this.drawRegion.show(new DrawView({
-            model: this.model
-        }));
-        this.placeMarkerRegion.show(new PlaceMarkerView({
-            model: this.model
-        }));
-        this.resetRegion.show(new ResetDrawView({
-            model: this.model
-        }));
-        this.streamRegion.show(new StreamSliderView({
-            model: this.model
-        }));
+        var draw_tools = settings.get('draw_tools');
+        if (_.indexOf(draw_tools, 'SelectArea') >= 0) {
```

Comment:
Not too much shorter, but `_.contains` may work better here.

http://underscorejs.org/#contains